packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -96,43 +117,45 @@ export const getComponentCss = (
     //     display: block,
     //   }
     header: {
-      display: 'grid',
-      gap: gridGap,
       padding: wrapContent ? `0 ${gridSafeZone}` : null,
       [mediaQueryS]: {
-        gridTemplateColumns: `minmax(0px, 1fr) ${pxToRemWithUnit(80)}`, // 2nd row has width of nav buttons
         position: 'relative',
-        minHeight: pxToRemWithUnit(40), // actual height of prev/next buttons
+        minHeight: '40px', // actual height of prev/next buttons
       },
     },
     nav: {
       display: 'none',
       [mediaQueryS]: {
         display: 'grid',
         gridAutoFlow: 'column',
+        gap: spacingStaticXSmall,
```

Comment:
Is it intentional that the right button goes further than the cut off slide?
<img width="215" alt="Bildschirm­foto 2023-02-06 um 10 57 30" src="https://user-images.githubusercontent.com/4380295/216941921-6e85f54a-da41-4ea5-93af-ca6bc90e2541.png">

Reply:
Fixed it